test/cql-pytest/test_compactionstats.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import pytest
+
+import nodetool
+from util import new_test_table
+
+
+def get_compaction_stats(cql, table):
+    ks, cf = table.split('.')
+    stats = nodetool.compactionstats(cql)
+    tasks = 0
+    for s in stats:
+        if s['ks'] == ks and s['cf'] == cf:
+            tasks += int(s['task'])
+    return tasks, stats
+
+@pytest.mark.parametrize("compaction_strategy", ["LeveledCompactionStrategy", "SizeTieredCompactionStrategy", "TimeWindowCompactionStrategy"])
+def test_compactionstats_after_major_compaction(scylla_only, cql, test_keyspace, compaction_strategy):
```

Comment:
You need to explain the reason for every use of "scylla_only" (in a docstring, if that's your poison, I don't understand what people have against comments).
Is it because Scylla's "nodetool compactionstats" is different? Why is it different? Is it deliberate?

Alternatively, if as @cezarmoise suggested, this test has nothing to do with nodetool and actually only checks the REST API, let's drop the nodetool facade and just say we are testing the REST API - and then obviously it will be scylla_only.

Reply:
I'll take the 2nd path of using only the rest api.
nodetool compactionstats semantics are not very well defined and I don't know what exactly to expect from cassandra.
The test assumes enough about the scylla implementation (like LCS estimated compaction tasks logic) to be limted to scylla only.